.gitignore

```diff
@@ -35,7 +35,6 @@ Testing/
 # Local-only config options
 configured.bazelrc
 user.bazelrc
-version_info.json
```

Comment:
Want to exclude `version_rc_info.json` now, so developers can test the `gen_version_info_rc.py` locally without polluting their git index?

We could also keep `version_info.json` ignored and choose a different name for the new files. I don't really like removing entries from `.gitignore` since switching branches during the transition period from ignored to unignored can be confusing.